wordcount.py

```diff
@@ -39,16 +39,34 @@
 
 import sys
 
-# +++your code here+++
-# Define print_words(filename) and print_top(filename) functions.
-# You could write a helper utility function that reads a file
-# and builds and returns a word/count dict for it.
-# Then print_words() and print_top() can just call the utility function.
+def print_words(filename):
```

Comment:
Good Python code is considered to generally respect the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/). To see what problems has your code, I've installed the `pep8` [library](https://pypi.python.org/pypi/pep8/1.6.2).